src/vs/workbench/browser/parts/views/views.ts

```diff
@@ -320,26 +320,17 @@ export class ContributableViewsModel extends Disposable {
 	}
 
 	private compareViewDescriptors(a: IViewDescriptor, b: IViewDescriptor): number {
-		const viewStateA = this.viewStates.get(a.id);
```

Comment:
@sandy081 FYI on this change to `compareViewDescriptors`

From @ryenus,

> return Number.MAX_VALUE instead of Number.POSITIVE_INFINITY for
views end up with undefined order, when comparing two views like this,
we get Number.MAX_VALUE - Number.MAX_VALUE = 0, rather than NaN
caused by infinity arithmetic.

Reply:
LGTM